src/plugins/DevTools/DevTools.css

```diff
@@ -55,6 +55,25 @@ devtools-panel .selectable-line.selected {
     background-color: rgba(118, 82, 108, 0.3);
 }
 
+devtools-panel button.logger {
+    float: right;
+    border: none;
+    background-color: transparent;
+    color: #666666;
+    padding: 0;
+    margin: 0;
+    display: none;
+}
+
+devtools-panel .about > button.logger {
+    display: inline;
+    padding-left: 1em;
```

Comment:
Didn't we discuss to use margin here rather than padding? Not a big deal though.